core/state_processor.go

```diff
@@ -403,9 +403,15 @@ func (p *StateProcessor) Process(block *types.Block, statedb *state.StateDB, cfg
 	posa, isPoSA := p.engine.(consensus.PoSA)
 	commonTxs := make([]*types.Transaction, 0, txNum)
 
-	// initilise bloom processors
+	// initialise bloom processors
 	bloomProcessors := NewAsyncReceiptBloomGenerator(txNum)
 	statedb.MarkFullProcessed()
+	signer := types.MakeSigner(p.config, header.Number)
+	// do trie prefetch for the big state trie tree in advance based transaction's From/To address.
+	go func() {
```

Comment:
I think it is not proper to do prefetch in `state_processor`,  it is better to follow the same style as snap prefetcher: do it on `insertChain`.